tests_and_analysis/test/script_tests/test_powder_map.py

```diff
@@ -43,7 +43,11 @@
      *quick_calc_params],
     [graphite_fc_file, '--asr', *quick_calc_params],
     [graphite_fc_file, '--asr=realspace', '--dipole-parameter=0.75',
-     *quick_calc_params]]
+     *quick_calc_params],
+    [graphite_fc_file, '--e-i=15', '--e-max=20', '--ebins=50',
+     *quick_calc_params],
```

Comment:
To fully cover the `if args.e_i is not None and args.e_max is None:...` branches we should also have a test where `--e-i` is set without `--e-max` 😢